### PR TITLE
fix/handlers

### PR DESCRIPTION
**Pull request by akash3gtm**
Added checks for POST data to be correct and in order

### CODE REVIEW

File: packages/cli/template/src/handlers/actions.js [hunk 1]
```diff
@@ -9,13 +9,20 @@
 const { DaprClient, HttpMethod } = require('@dapr/dapr');
 
 module.exports = async (req, res) => {
-  if (!req.params || !req.params.action_name) {
+  if ( !req.params || !req.params.action_name ) {
     return res.status(500).json({
       status: false,
       message: '"action" is missing from request param'
```
Comment (faizahmedfarooqui) [inline packages/cli/template/src/handlers/actions.js]: Correction: "action_name" is missing

File: packages/cli/template/src/handlers/actions.js [hunk 2]
```diff
@@ -9,13 +9,20 @@
...
     });
   }
 
+  if ( !req.body.action ) {
+    return res.status(500).json({
+      status: false,
+      message: '"action" is missing from request data'
```
Comment (faizahmedfarooqui) [inline packages/cli/template/src/handlers/actions.js]: Correction: "action" is missing from request body

**Comment by faizahmedfarooqui** [comment]
Update the package version to the latest and recreate the typescript build.

**Comment by faizahmedfarooqui** [comment]
@akash3gtm I have added 3 comments above, please fix them and let me know once this is done.

File: packages/cli/template/src/handlers/actions.js (hunk 2 above)
Comment (akash3gtm) [reply packages/cli/template/src/handlers/actions.js]: updated in latest commit

File: packages/cli/template/src/handlers/actions.js (hunk 1 above)
Comment (akash3gtm) [reply packages/cli/template/src/handlers/actions.js]: updated in latest commit

**Comment by akash3gtm** [comment]
Fixed all 3 issues and pushed them to the repo.
Please review and approve.

**Review by faizahmedfarooqui — APPROVED** [review]

**Pull request merged**